server/src/main/java/org/opensearch/indices/IndicesRequestCache.java

```diff
@@ -103,13 +109,16 @@ public final class IndicesRequestCache implements RemovalListener<IndicesRequest
         Property.NodeScope
     );
 
+    private final static long BASE_RAM_BYTES_USED = RamUsageEstimator.shallowSizeOfInstance(Key.class);
+
     private final ConcurrentMap<CleanupKey, Boolean> registeredClosedListeners = ConcurrentCollections.newConcurrentMap();
     private final Set<CleanupKey> keysToClean = ConcurrentCollections.newConcurrentSet();
     private final ByteSizeValue size;
     private final TimeValue expire;
     private final Cache<Key, BytesReference> cache;
+    private final Function<ShardId, CacheEntity> cacheEntityFunction;
```

Comment:
Give this a more descriptive name -- like `CacheEntityRetriever` or `CacheEntityLookup`.

Reply:
Changed to ```CacheEntityLookup```